psd-web/spec/features/password_reset_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+
+RSpec.describe "Password reset management", :with_test_queue_adpater do
+  let(:user)                         { create(:user) }
+  let!(:reset_token)                 { Devise.token_generator.generate(User, :reset_password_token) }
+  let(:edit_user_password_url_token) { edit_user_password_url(reset_password_token: reset_token.first) }
+
+
+  def send_reset_password
```

Comment:
```suggestion
  def request_password_reset
```
Seems to describe more accurately what is happening.